clients/js/package.json

```diff
@@ -69,7 +69,8 @@
       "rewritePaths": {
         "test/": "dist/test/"
       }
-    }
+    },
+    "timeout": "60s"
```

Comment:
Tests should work without increasing timeouts.

Reply:
Alright thank you for your feedback , will fix these and get back to you right away

Reply:
Also ran formatting , can i rebase now? or theres any other thing I need to modify